commons-core/src/main/scala/com/avsystem/commons/rpc/RpcUtils.scala

```diff
@@ -0,0 +1,20 @@
+package com.avsystem.commons
+package rpc
+
+import com.avsystem.commons.macros.misc.MiscMacros
+
+/**
+  * @author ghik
+  */
+object RpcUtils {
```

Comment:
These are only used in macro-generated code - maybe we could move this to the macro module?